styleguide/Components/Icon28TokenizedOutline/Icon28TokenizedOutline.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+export const Icon28TokenizedOutline = ({
+  className,
+  width = 28,
+  height = 28,
+  stroke,
+  title,
+}) => {
+  return (
+    <div
+      role="presentation"
```

Comment:
```suggestion
  title,
  getRootRef,
}) => {
  return (
    <div
      ref={getRootRef}
      role="presentation"
```
Сломалось наведение на иконку, нужно добавить `getRootRef`
<img width="533" alt="image" src="https://user-images.githubusercontent.com/14944123/179026514-d8c80c83-9a02-4b3c-80bf-3a8ec00a9318.png">